### DISCUSSION TITLE
Unclear Existence of RAM Limit Default

### DISCUSSION THREAD

**Discussion by DarioS** [PGScatalog/pgsc_calc#257] [Q&A] [answered]
### Description of the bug

> The workflow isn’t using many resources (e.g. RAM / CPU)
> Did you forget to set `--max_cpu` or `--max_memory`?

It is unclear if there is any limit by default or it will consume as much as necessary. If there is, it should be explicitly stated.

### Relevant files

Troubleshooting section of User Guide.

**Comment by smlmbrt** [comment, marked as answer]
The defaults are set in the [`nextflow.config `](https://github.com/PGScatalog/pgsc_calc/blob/main/nextflow.config) file, and they currently are:
```
    // Max resource options
    // Defaults only, expecting to be overwritten
    max_memory                 = '16.GB'
    max_cpus                   = 2
    max_time                   = '240.h'
```
I would just recommend setting these to whatever your containers allow (e.g. docker resource limits on a laptop), or for your compute cluster (see https://pgsc-calc.readthedocs.io/en/latest/troubleshooting.html#the-workflow-isn-t-using-many-resources-e-g-ram-cpu).
